pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -7,9 +7,10 @@
         >>
   %div.announce-content
     %div.content-img-container
-      %img.content-img{:src=>"/shared/images/courses/fit-x108/logo_tall_playlab.jpg"}
+      %img.content-img{:src=>"/images/fit-x108/hour-of-code-logo.png"}
     %div.content-text
-      =I18n.t(:dashboard_announce_playlab)
+      %b December 8, 2015
+      Due to the extraordinary number of users visiting our site for the Hour of Code, saving course progress has been temporarily suspended. We are working hard to restore this functionality. Thank you for your patience during this exciting time.
   %div.announce-learn-more
-    %a{:href=>"/learn"}
-      %button!= I18n.t(:learn_more)
+    %a{:href=>"https://studio.code.org/"}
```

Comment:
we should use CDO stuff here so that this works correctly in each environment (localohost/staging/test/etc)